x-pack/test/security_solution_cypress/es_archives/auditbeat/data.json

```diff
@@ -83,7 +83,12 @@
         "args" : [
           "-zsh"
         ],
-        "entity_id" : "q6pltOhTWlQx3BCD"
+        "entity_id" : "q6pltOhTWlQx3BCD",
```

Comment:
Really good approach adding the data needed instead of creating a new file. Thanks! :)